docs/reference/manifest/queryuri.md

```diff
@@ -1,3 +1,9 @@
+---
+title: QueryUri element in the manifest file
```

Comment:
Entirely separate from this checkin, but why is this element a "Uri" while the others are "Urls" The subsequent text description also says "URL".

Reply:
Hmmm...I'm not sure about the history of this particular element re naming and description. But, given that URL is a subset of URI (i.e., something that's a URL is also a URI), I don't think it's necessarily inaccurate as currently written.

Reply:
True, though I'd expect the description to also use "URI", not "URL".